plugins/rebalance/eventnotify.go

```diff
@@ -0,0 +1,80 @@
+package rebalance
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+
+	log "github.com/sirupsen/logrus"
+)
+
+//HandleEventNotify updates the rebalinfo in the store with the status sent by
+// the rebalance process
+func HandleEventNotify(status map[string]string) error {
+	var (
+		ok      bool
+		volname string
+		err     error
+	)
+
+	log.Info("[nithya]: In RebalanceHandleEventNotify ")
```

Comment:
You should either remove these temp logs, or better convert them into actual logs. Debug logs should be fine for this.

Reply:
I missed this. Will remove.